Tasks/CocoaPodsV0/task.json

```diff
@@ -17,8 +17,8 @@
     "minimumAgentVersion": "2.182.1",
     "version": {
         "Major": 0,
-        "Minor": 201,
-        "Patch": 1
+        "Minor": 207,
```

Comment:
you plan to merge this only next week? We are still in 206.

Reply:
I just refer to [this](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) guide, since Tuesday is passed I used current sprint + 1